spec/requests/api/journalist/journalist_can_create_articles_spec.rb

```diff
@@ -1,6 +1,9 @@
 RSpec.describe 'POST /api/articles', type: :request do
   let(:journalist) { create(:user, role: 'journalist') }
   let(:auth_headers) { journalist.create_new_auth_token }
+  let(:image) do
+    [ File.read(fixture_path + '/fake-news-fixture.txt') ]
```

Comment:
Same here with the array, can remove

Reply:
and this

Reply:
Nah.. doesnt work as a string. Gives this error ActiveSupport::MessageVerifier::InvalidSignature when we create article.
You will have to send an array to us